spec/cordova/plugin/add.spec.js

```diff
@@ -381,15 +381,15 @@ describe('cordova/plugin/add', function () {
                     });
             });
             it('should retrieve installed plugins and installed platforms version and feed that information into determinePluginVersionToFetch', function () {
-                plugin_util.getInstalledPlugins.and.returnValue([{ 'id': 'cordova-plugin-camera', 'version': '2.0.0' }]);
+                plugin_util.getInstalledPlugins.and.returnValue([{ 'id': '@cordova/plugin-thinger', 'version': '2.0.0' }]);
```

Comment:
Do we need to continue to test a plugin that is not scoped?

Reply:
I also thought about that. I actually had it that way before. I didn't particularly like it, because it basically duplicated the test. I thought if a scoped plugin would work, then a normal one should work too. But that implication is not necessarily true of course 🤷‍♂

Reply:
I agree with @raphinesse here, while in theory if scope works, then non-scoped plugins should work as well, but they are two different formats and I don't think it hurts testing both formats for peace of mind.

Reply:
I took another look at the tests and what was being tested and I don't really see the value of duplicating the test here.